src/support_sphere/lib/presentation/pages/main_app/profile/profile_body.dart

```diff
@@ -200,14 +194,16 @@ class _HouseholdInformation extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final formKey = GlobalKey<FormBuilderState>();
+
     return BlocBuilder<ProfileCubit, ProfileState>(
       buildWhen: (previous, current) => previous.household != current.household,
       builder: (context, state) {
         Household? household = state.household;
         String address = household?.address ?? '';
         String pets = household?.pets ?? '';
         String notes = household?.notes ?? '';
-        String accessibilityNeeds = household?.accessibility_needs ?? 'None';
+        String accessibilityNeeds = household?.accessibility_needs ?? '';
```

Comment:
```suggestion
        String accessibilityNeeds = household?.accessibility_needs ?? 'Not Applicable';
```

I think it makes more sense for a person without any accessibility needs to be Not Applicable.. probably want to put that default string into the strings catalog.